src/main/kotlin/se/inera/intyg/IntygPlugin.kt

```diff
@@ -228,6 +228,7 @@ class IntygPlugin : Plugin<Project> {
 
             val dependencyCheckExtension = project.extensions.getByType(DependencyCheckExtension::class.java)
             dependencyCheckExtension.formats = listOf("HTML", "JSON")
+            dependencyCheckExtension.nvd.datafeedUrl = "https://vulnz-nvd-mirror-devtest.intyg.nordicmedtest.se"
```

Comment:
I think it would be a good idea to make this configurable. 

Maybe do something like this... if a datafeedUrl is provided, we set it. Otherwise it is left unchanged and default configuration will be used.

What do you think?

Reply:
Good idea, I'll have a look at it...